### PR TITLE
New RUM Platform section

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?
- Adds a new Platform H2
- Nests Dashboards, Monitors, Generate Custom Metrics, and Connect RUM and Traces
- Updates aliases

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by aliciascott** [comment]
Docs [card](https://datadoghq.atlassian.net/browse/DOCS-7415) for review

File: config/_default/menus/menus.en.yaml [hunk 1]
```diff
@@ -4622,146 +4622,156 @@ main:
     parent: rum_session_replay
     identifier: rum_session_replay_playlists
     weight: 305
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4624]:
```suggestion
    weight: 405
```

File: config/_default/menus/menus.en.yaml [hunk 2]
```diff
@@ -4622,146 +4622,156 @@ main:
...
-  - name: Heatmaps
-    url: real_user_monitoring/heatmaps
+  - name: Platform
+    identifier: real_user_monitoring_platform
+    url: real_user_monitoring/platform
     parent: rum
-    identifier: rum_heatmaps
     weight: 4
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4629]: I think this is supposed to be 5, unless you want this above Session Replay
```suggestion
    weight: 5
```

File: config/_default/menus/menus.en.yaml [hunk 3]
```diff
@@ -4622,146 +4622,156 @@ main:
...
   - name: Dashboards
-    url: real_user_monitoring/dashboards/
-    parent: rum
+    url: real_user_monitoring/platform/dashboards/
+    parent: rum_platform
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4632]:
```suggestion
    parent: real_user_monitoring_platform
```

File: config/_default/menus/menus.en.yaml [hunk 4]
```diff
@@ -4622,146 +4622,156 @@ main:
...
     identifier: rum_dashboards
-    weight: 5
+    weight: 401
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4634]:
```suggestion
    weight: 501
```

File: config/_default/menus/menus.en.yaml [hunk 5]
```diff
@@ -4622,146 +4622,156 @@ main:
...
+    url: real_user_monitoring/platform/dashboards/errors
     parent: rum_dashboards
     identifier: rum_dashboards_errors
-    weight: 504
+    weight: 104
+  - name: Monitors
+    url: monitors/types/real_user_monitoring/
+    parent: real_user_monitoring_platform
+    identifier: rum_monitors
+    weight: 402
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4659]:
```suggestion
    weight: 502
```

File: config/_default/menus/menus.en.yaml [hunk 6]
```diff
@@ -4622,146 +4622,156 @@ main:
...
+  - name: Generate Custom Metrics
+    url: real_user_monitoring/generate_metrics
+    parent: real_user_monitoring_platform
+    identifier: rum_generate_metrics
+    weight: 403
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4664]:
```suggestion
    weight: 503
```

File: config/_default/menus/menus.en.yaml [hunk 7]
```diff
@@ -4622,146 +4622,156 @@ main:
...
+  - name: Connect RUM and Traces
+    url: real_user_monitoring/connect_rum_and_traces
+    parent: real_user_monitoring_platform
+    identifier: rum_connect_rum_and_traces
+    weight: 404
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:4669]:
```suggestion
    weight: 504
```

**Review by estherk15 — APPROVED** [review]
Looks good! There's a shift in the menu, depending on where you want the new Platform header to be.

**Pull request merged**